test/integration/targets/azure_rm_virtualmachine/tasks/setup.yml

```diff
@@ -0,0 +1,74 @@
+- name: Create random names
```

Comment:
not necessary to separate a setup file, but it's no harm.

Reply:
Seems like it would be reusable so it makes sense to keep it separate.